libs/langchain/langchain/llms/gradient_ai.py

```diff
@@ -41,7 +44,7 @@ class GradientLLM(LLM):
 
     """
 
-    model_id: str
```

Comment:
for backwards compatibility we could alias this so you could specify as either model or model_id. this is a new enough integration that it's probably ok to not do that, will let you decide what's best